src/Aspire.Hosting/Dcp/DcpExecutor.cs

```diff
@@ -131,7 +131,7 @@ public DcpExecutor(ILogger<DcpExecutor> logger,
     }
 
     private string ContainerHostName => _configuration["AppHost:ContainerHostname"] ??
-        (_options.Value.EnableAspireContainerTunnel ? KnownHostNames.DefaultContainerTunnelHostName : KnownHostNames.DockerDesktopHostBridge);
+        (_options.Value.EnableAspireContainerTunnel ? KnownHostNames.DefaultContainerTunnelHostName : _dcpInfo?.Containers?.HostName ?? KnownHostNames.DockerDesktopHostBridge);
```

Comment:
Isn't `dcpInfo.Containers.HostName` kind of useless? I mean, with Podman, it will result in a name that maps to Podman host VM name, and not the localhost network for the Podman host OS, no?

Reply:
That is true, but my point is either way it is not going to work as intended.